website/docs/r/securityhub_insight.html.markdown

```diff
@@ -0,0 +1,388 @@
+---
+subcategory: "Security Hub"
+layout: "aws"
+page_title: "AWS: aws_securityhub_insight"
+description: |-
+  Provides a Security Hub custom insight resource.
+---
+
+# Resource: aws_securityhub_insight
+
+Provides a Security Hub custom insight resource. See the [Managing custom insights section](https://docs.aws.amazon.com/securityhub/latest/userguide/securityhub-custom-insights.html) of the AWS User Guide for more information.
+
+## Example Usage
+
+### Filter by AWS account ID
+
+```terraform
+resource "aws_securityhub_account" "example" {}
+
+resource "aws_securityhub_insight" "example" {
+  filters {
+    aws_account_id {
+      comparison = "EQUALS"
+      value      = "1234567890"
+    }
+
+    aws_account_id {
+      comparison = "EQUALS"
+      value      = "09876543210"
+    }
+  }
+
+  group_by_attribute = "AwsAccountId"
+
+  name = "example-insight"
+
+  depends_on = [aws_securityhub_account.example]
+}
+```
+
+### Filter by date range
+
+```terraform
+resource "aws_securityhub_account" "example" {}
+
+resource "aws_securityhub_insight" "example" {
+  filters {
+    created_at {
+      date_range {
+        unit  = "DAYS"
+        value = 5
+      }
+    }
+  }
+
+  group_by_attribute = "CreatedAt"
+
+  name = "example-insight"
+
+  depends_on = [aws_securityhub_account.example]
+}
+```
+
+### Filter by destination IPv4 address
+
+```terraform
+resource "aws_securityhub_account" "example" {}
+
+resource "aws_securityhub_insight" "example" {
+  filters {
+    network_destination_ipv4 {
+      cidr = "10.0.0.0/16"
+    }
+  }
+
+  group_by_attribute = "NetworkDestinationIpV4"
+
+  name = "example-insight"
+
+  depends_on = [aws_securityhub_account.example]
+}
+```
+
+### Filter by finding's confidence
+
+```terraform
+resource "aws_securityhub_account" "example" {}
+
+resource "aws_securityhub_insight" "example" {
+  filters {
+    confidence {
+      gte = "80"
+    }
+  }
+
+  group_by_attribute = "Confidence"
+
+  name = "example-insight"
+
+  depends_on = [aws_securityhub_account.example]
+}
+```
+
+### Filter by resource tags
+
+```terraform
+resource "aws_securityhub_account" "example" {}
+
+resource "aws_securityhub_insight" "example" {
+  filters {
+    resource_tags {
+      comparison = "EQUALS"
+      key        = "Environment"
+      value      = "Production"
+    }
+  }
+
+  group_by_attribute = "ResourceTags"
+
+  name = "example-insight"
+
+  depends_on = [aws_securityhub_account.example]
+}
+```
+
+## Argument Reference
+
+The following arguments are required:
+
+* `filters` - (Required) A configuration block including one or more (up to 10 distinct) attributes used to filter the findings included in the insight. The insight only includes findings that match criteria defined in the filters. See below for more details.
+
```

Comment:
```suggestion
```

I think it's more common to not have a line between arguments. At least one benefit to no lines between is you can select all the lines and sort them in vscode.